Bixal/move.mil
# Feature 156885931 include node class module

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- includes node_class module in our project
- includes patch for known issue with https://www.drupal.org/project/node_class/issues/2907451
- includes config to enable the module
- includes styles to associate with first node_class instance **svc-two-col-content**

## Testing

To verify the changes proposed in this pull request…

1. pull changes locally
2. make composer
3. make cr
4. cd /move-mil/web/themes/custom/move_mil
5. make build
6. create a basic page
7. add **svc-two-col-content** to node class field in node admin pannel

![screen shot 2018-04-18 at 12 10 12 pm](https://user-images.githubusercontent.com/37077057/38945222-f52c9e64-4303-11e8-9152-0c8fd3bf64ed.png)
![screen shot 2018-04-18 at 12 10 01 pm](https://user-images.githubusercontent.com/37077057/38945227-f780ec92-4303-11e8-92d3-d842fbf0a1a3.png)

8. add callout accordion paragraph in the first paragraph order
9. add customer-service-paragraph(s) as needed
10. add alert information paragraph in the second paragraph order
11. add basic text in the third paragraph order
12. in the body content add the army banner
![army-banner-594a898aaaa5083bdfdfcc8e1693bc4e800c8429c237253e2cbb7bee067d2e96](https://user-images.githubusercontent.com/37077057/38945315-2ece83f8-4304-11e8-8fea-370adbeadf8f.jpg)

save and view, you should see be able to create the following

![customerserviceinfoparagraph](https://user-images.githubusercontent.com/37077057/38945347-4385cf68-4304-11e8-83e4-e945fafcad8d.gif)
